### ISSUE TITLE
Unable to install @angular/cli node-npm is trowing error checksum failed while installing.

### ISSUE THREAD

**Issue by isnegi** [npm/npm#18992]
I'm using node v6.11.5 and npm 3.10.10 and 64-bit OS, and getting some error while installing @angular/cli.

Each time, this shasum check failed is all I'm getting. I tried the following things:

1. I configured the npm registry to both http and https. Didn't worked.
2. Checked the path OS path variables, and the directory where node is installed. Found nothing.
3. Tried to clean npm and uninstalling and installing angular/cli. Didn't worked.
See the attached log file for more info.
[npm-debug.log](https://github.com/npm/npm/files/1424263/npm-debug.log)

**Comment by isnegi** [comment]
I've tried installing different versions of node, I thought the recent version might be having some issues. But it didn't helped.

**Comment by isnegi** [comment]
@KenanY How to add label to the issue. Please add attach label "npm" as well. It might be because of some npm configurations.

**Comment by kenany** [comment]
@shubham-negi We currently do not have such a label. Since this issue tracker is for `npm`-specific issues only, there is no need for a `npm` label.

**Comment by isnegi** [comment]
@KenanY Till when can I suppose to getting this work.? I've deadline on angular work and still haven't configured the project. ;(

**Comment by kenany** [comment]
Unfortunately I do not know where the root cause of your issue lies, so I cannot provide any support 😢

**Comment by shubham-negi** [comment]
@KenanY Well let me know when you're up for something, I can provide you the information.

**Comment by mikefitz-tx** [comment]
Did you ever find a solution to this problem? I am experiencing the same thing, and my error log looks just like yours.

**Comment by isnegi** [comment]
@mikefitz-tx Yes, and that is try some other machine. ;) Why things happens strange sometimes. You're working on something interesting @mikefitz-tx ?
